Frameworks/Foundation/NSMutableURLRequest.mm

```diff
@@ -76,4 +84,18 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     return [self mutableCopyWithZone:zone];
 }
 
+- (void)_setProperty:(id)value forKey:(NSString*)key {
+    @synchronized(self) {
+        [self _lazyInitProperties];
+        [_properties setValue:value forKey:key];
```

Comment:
Method has nonnull declaration on ref plat, but acts the same as remove if forced, which is already handled by the NSMutableDictionary side of things.